### PR TITLE
add workflow vsphere-upi-disconnected-sdn-workers-rhcos-rhel8

### PR DESCRIPTION
**Pull request by lihongan**

### CODE REVIEW

**Comment by lihongan** [comment]
@jinyunma  @jianlinliu  Could you help take a look the error ?
```
 level=error msg=Bootstrap failed to complete: Get "https://api.ci-op-6r69m3m0-f0a0b.vmc-ci.devcluster.openshift.com:6443/version": dial tcp 192.168.101.2:6443: connect: connection refused
level=error msg=Failed waiting for Kubernetes API. This error usually happens when there is a problem on the bootstrap host that prevents creating a temporary control plane. 
```

**Comment by jinyunma** [comment]
> @jinyunma @jianlinliu Could you help take a look the error ?
> 
> ```
>  level=error msg=Bootstrap failed to complete: Get "https://api.ci-op-6r69m3m0-f0a0b.vmc-ci.devcluster.openshift.com:6443/version": dial tcp 192.168.101.2:6443: connect: connection refused
> level=error msg=Failed waiting for Kubernetes API. This error usually happens when there is a problem on the bootstrap host that prevents creating a temporary control plane. 
> ```
@lihongan for disconnected profile, cluster_profile should use `vsphere-discon`.

**Comment by lihongan** [comment]
/assign @jianlinliu 
The tests and installed cluster look good now. 
cc @zhaozhanqi @anuragthehatter

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml [hunk 1]
```diff
@@ -67,8 +67,8 @@ tests:
 - as: installer-rehearse-vsphere
   interval: 960h
   steps:
-    cluster_profile: vsphere
-    workflow: cucushift-installer-rehearse-vsphere-ipi-sdn
+    cluster_profile: vsphere-discon
```
Comment (jianlinliu) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml:70]: How about adding a dedicated test for vsphere-discon cluster_profile in the 4.12.yaml just like what we did in 4.10

Comment (jianlinliu) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml:70]: But it is okay for this PR. We can file another PR to follow up that.

Comment (lihongan) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml:70]: Good suggestion! We can add that in next PR, just keep this one for disconnected profile and add another one for non-disconnected.

**Pull request merged**